wazuh/README.md

```diff
@@ -0,0 +1,155 @@
+
+3. [Restart the Agent][2].
+
+#### Configure syslog message forwarding from Wazuh
+
+  1. Log in to the Wazuh UI. Navigate to the Left side Menu.
+  2. Go to **Server management** > **Settings**.
+  3. Click on **Edit configuration**.
+  4. Add the following configuration block:
+
+      In this example, all alerts are sent to 1.1.1.1 on port 8080 in JSON format.
+      ```xml
+        <syslog_output>
+          <server>1.1.1.1</server>
+          <port>8080</port>
+          <format>json</format>
+        </syslog_output>
+      ```
+
+      * The `server` tag should contain the IP address where your Datadog Agent is running.
+
+      * The `port` tag should contain the port on which your Datadog Agent is listening.
+
+      Note: Using JSON format is required, since Wazuh pipeline parses JSON formatted logs only.
+  5. Click the **Save** button.
+  6. After saving, Click on the **Restart Manager** button.
+
+
+### Validation
+
+[Run the Agent's status subcommand][5] and look for `wazuh` under the Checks section.
+
+## Data Collected
+
+### Log 
+
+| Format     | Event Types    |
+| ---------  | -------------- |
+| JSON | vulnerability-detector, file-integrity-monitoring, malware-detector, github, docker, amazon, office365, google-cloud, system and other |
+
+### Metrics
+
+The Wazuh integration does not include any metrics.
+
+### Events
+
+The Wazuh integration does not include any events.
+
+### Service Checks
+
+The Wazuh integration does not include any service checks.
+
+## Troubleshooting
+
+**Permission denied while port binding:**
+
+If you see a **Permission denied** error while port binding in the Agent logs:
+
+1. Binding to a port number under 1024 requires elevated permissions. Grant access to the port using the `setcap` command:
+    ```shell
+    sudo setcap CAP_NET_BIND_SERVICE=+ep /opt/datadog-agent/bin/agent/agent
+    ```
+
+2. Verify the setup is correct by running the `getcap` command:
+
+    ```shell
+    sudo getcap /opt/datadog-agent/bin/agent/agent
+    ```
+
+    With the expected output:
+
+    ```shell
+    /opt/datadog-agent/bin/agent/agent = cap_net_bind_service+ep
+    ```
+
+    **Note**: Re-run this `setcap` command every time you upgrade the Agent.
+
+3. [Restart the Agent][2].
+
+**Data is not being collected:**
+
+Make sure that traffic is bypassed from the configured port if the firewall is enabled.
+
+**Port already in use:**
+
+- If you see the **Port <PORT_NUMBER> Already in Use** error, see the following instructions. The example below is for port 514:
+
+- On systems using Syslog, if the Agent listens for Wazuh logs on port 514, the following error can appear in the Agent logs: `Can't start UDP forwarder on port 514: listen udp :514: bind: address already in use`. This error occurs because by default, Syslog listens on port 514. To resolve this error, take **one** of the following steps: 
+
+    - Disable Syslog.
+    - Configure the Agent to listen on a different, available port.
```

Comment:
```suggestion

Here is how to troubleshoot some possible issues.

**Data is not being collected:**

Ensure traffic is bypassed from the configured port if the firewall is enabled.

**Port already in use:**

If you see the **Port <PORT_NUMBER> Already in Use** error, see the following instructions. The example below is for port 514:

- On systems using Syslog, if the Agent listens for Wazuh logs on port 514, the following error can appear in the Agent logs: `Can't start UDP forwarder on port 514: listen udp :514: bind: address already in use`. This error occurs because by default, Syslog listens on port 514. To resolve this error, take **one** of the following steps: 
    - Disable Syslog.
    - Configure the Agent to listen on a different, available port.
```

Reply:
Thank you for the suggestion. We’ve made the updates accordingly, but we believe the line `Here is how to troubleshoot some possible issues` might be better placed at the top of the troubleshooting section as it applies to all the potential issues users might encounter including `Permission denied while port binding:`